example/lib/main.dart

```diff
@@ -20,72 +19,72 @@ final Dio dio = Dio()..options = BaseOptions(connectTimeout: 10000);
 
 final navigatorKey = GlobalKey<NavigatorState>();
 
-void main() {
-  if (kDebugMode) {
-    PluginManager.instance
-      ..register(WidgetInfoInspector())
-      ..register(WidgetDetailInspector())
-      ..register(ColorSucker())
-      ..register(AlignRuler())
-      ..register(ColorPicker())
-      ..register(TouchIndicator())
-      ..register(Performance())
-      ..register(ShowCode())
-      ..register(MemoryInfoPage())
-      ..register(CpuInfoPage())
-      ..register(DeviceInfoPanel())
-      ..register(Console())
-      ..register(DioInspector(dio: dio))
-      ..register(CustomRouterPluggable())
-      ..register(ChannelPlugin());
-    runApp(
-      MultiProvider(
-        providers: [
-          ChangeNotifierProvider(create: (_) => UMESwitch()),
-        ],
-        builder: (ctx, child) => UMEWidget(
-          enable: ctx.watch<UMESwitch>().enable,
-          child: MyApp(),
-        ),
-      ),
-    );
-  } else {
-    runApp(MyApp());
-  }
-}
+void main() => runApp(const UMEApp());
+
+class UMEApp extends StatefulWidget {
+  const UMEApp({Key? key}) : super(key: key);
 
-class MyApp extends StatefulWidget {
   @override
-  State<MyApp> createState() => _MyAppState();
+  State<UMEApp> createState() => _UMEAppState();
 }
 
-class _MyAppState extends State<MyApp> {
+class _UMEAppState extends State<UMEApp> {
   @override
   void initState() {
     super.initState();
-    WidgetsBinding.instance?.addPostFrameCallback((_) {
-      CustomRouterPluggable().navKey = navigatorKey;
```

Comment:
It's the global navigator key. `CustomRouterPluggable` need it and build the "Anywhere door".

Reply:
We should keep the line. And find an appropriate time to set it.

Reply:
Oh it's an accident. Will add it back later.